locking/urls.py

```diff
@@ -6,6 +6,8 @@
 
 __all__ = ('urlpatterns', )
 
+app_name = 'locking'
```

Comment:
I follow this example from documentation https://docs.djangoproject.com/en/1.9/topics/http/urls/#id4

For example, specifying `app_name` in url conf  gives you ability to reverse urls in package like `reverse(locking:locking-api)`

and include package urls in project like:
```python
url(r'^locking/', include('locking.urls', namespace='lock')),
```
and reverse it `reverse('lock:locking-api')`.

But, unfortunately, this doesn't make sense in Django 1.8